src/test/scala/org/dbpedia/spotlight/wikistats/TestSuite.scala

```diff
@@ -104,12 +113,12 @@ class TestSuite extends FunSuite with SharedSparkContext with BeforeAndAfter{
       .collect()
       .toList
       .foreach(count => {
-           if (count == -1) annotatedCounts += 0
-           else annotatedCounts +=count
+      if (count == -1) annotatedCounts += 0
+      else annotatedCounts +=count
     })
 
     assert(totalLinks==annotatedCounts)
   }
 
 
-}
+}
```

Comment:
I don't know how this is coming. May be an intellij issue. And it is looking correct in my laptop.